### PR TITLE
chore: upgrade API spec to version 0.0.31

### PR DESCRIPTION
**Pull request by dgonzalez**
- API version: `1.327.0`
- Package version: `0.0.31`

Added package deny rules policy.

### CODE REVIEW

**Review by estebangarcia — APPROVED** [review]
🤖

File: bin/generate [hunk 1]
```diff
@@ -2,7 +2,7 @@
 set -eou pipefail
 
 
-PKG_VERSION=${1:-"0.0.14"}
+PKG_VERSION=${1:-"0.0.31"}
```
Comment (jackgibson1) [inline bin/generate:5]: Good catch

File: bin/generate [hunk 2]
```diff
@@ -54,17 +54,17 @@ rm -rfv test/
...
+gsed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token.go
+gsed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_request.go
+gsed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_request.go
+gsed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_request_patch.go
+gsed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_request_patch.go
+gsed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_refresh.go
+gsed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_refresh.go
+gsed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_refresh_request.go
+gsed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_refresh_request.go
+gsed -i 's/ACTION/string/g' model_package_tag_request.go
```
Comment (jackgibson1) [inline bin/generate:67]: Just curious, what's the advantage of GNU `sed` over the former?

**Review by jackgibson1 — APPROVED** [review]
model changes and additions seem sane to me!

File: bin/generate (hunk 2 above)
Comment (dgonzalez) [reply bin/generate:67]: no advantage, sed doesnt work with the mac version.

**Pull request merged**